# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/cinepaint/cinepaint-gcc4.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./app/histogram_tool.c.vanilla	2005-12-14 16:29:07.000000000 +0100
+++ ./app/histogram_tool.c	2005-12-14 16:31:31.000000000 +0100
@@ -84,7 +84,7 @@
 static void *histogram_tool_options = NULL;
 static HistogramToolDialog *histogram_tool_dialog = NULL;
 
-static Argument * histogram_invoker (Argument *args);
+Argument * histogram_invoker (Argument *args);
 
 static char * histogram_info_names[7] =
 {
@@ -538,7 +538,7 @@
 /*
  *  TBD - WRB -make work with float data
 */
-static Argument *
+Argument *
 histogram_invoker (Argument *args)
 {
   Argument *return_args;
--- ./plug-ins/bracketing_to_hdr/Fl_Cartesius.cpp.vanilla	2005-12-15 02:17:32.000000000 +0100
+++ ./plug-ins/bracketing_to_hdr/Fl_Cartesius.cpp	2005-12-15 02:33:44.000000000 +0100
@@ -804,7 +804,7 @@
   case FL_MOVE:
     mk_coord_picker_str(wx_from_x(Fl::event_x()), wy_from_y(Fl::event_y()));
     //draw_coord_picker();      // now via the damage() channel...
-    damage (FL_DAMAGE_USER1);   // my code word for "draw only coord picker"
+    //damage (FL_DAMAGE_USER1);   // my code word for "draw only coord picker"
     return 1;
   
   case FL_ENTER:
@@ -896,13 +896,14 @@
 
   // Check if only the coord picker is to update (before Fl_Group::draw(), 
   //  which clears all!) Note: draw_coord() doesn't need win-world-map... 
-  
+ 
+/* 
   if (damage() == FL_DAMAGE_USER1) { // &-question fails (MOVE-Problem)
     draw_coord_picker();            
     TRACE_DRAW(("only coord_picker\n"))
     return;
   }
-  
+*/ 
   if (rejustify_) justify();    // Before draw_children(), so that children 
                                 //  have the right map too
   
--- ./plug-ins/bracketing_to_hdr/fl_print_event.cpp.vanilla	2005-12-15 02:34:37.000000000 +0100
+++ ./plug-ins/bracketing_to_hdr/fl_print_event.cpp	2005-12-15 02:34:57.000000000 +0100
@@ -89,8 +89,8 @@
     if (mask & FL_DAMAGE_EXPOSE)  strcat(s," EXPOSE");
     if (mask & FL_DAMAGE_SCROLL)  strcat(s," SCROLL");
     if (mask & FL_DAMAGE_OVERLAY) strcat(s," OVERLAY");
-    if (mask & FL_DAMAGE_USER1)   strcat(s," USER1");
-    if (mask & FL_DAMAGE_USER2)   strcat(s," USER2");
+//    if (mask & FL_DAMAGE_USER1)   strcat(s," USER1");
+//    if (mask & FL_DAMAGE_USER2)   strcat(s," USER2");
     if (mask & FL_DAMAGE_ALL)     strcat(s," ALL"); 
   }
   printf("damage mask: %d: %s", mask, s);
